src/components/Button/Button.module.scss

```diff
@@ -33,10 +32,20 @@ $disabled-on-primary-color-opacity: 0.5;
       height: 100%;
       margin: 0;
     }
+  }
 
-    .textPlaceholder {
-      opacity: 0;
-    }
+  .textPlaceholder {
+    opacity: 0;
+    height: 0;
+    visibility: hidden;
+    white-space: pre;
+  }
+
+  .successContent {
+    position: absolute;
```

Comment:
This assumes that the text size of the success is shorter than the original content

Reply:
Good point, I flexed it instead, thanks!